src/libraries/Microsoft.PowerFx.Core/Public/Values/DateTimeValue.cs

```diff
@@ -15,6 +15,28 @@ namespace Microsoft.PowerFx.Types
     /// </summary>
     public class DateTimeValue : PrimitiveValue<DateTime>
```

Comment:
Is there something we can do like 'new' keyword on Value to block usage of that and force via the safe helper

Reply:
Added new keyword, to block usage of the field directly.